.teamcity/settings.kts

```diff
@@ -0,0 +1,88 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.commitStatusPublisher
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
+
+project {
+    buildType(ChocolateyGUI)
+}
+
+object ChocolateyGUI : BuildType({
+    id = AbsoluteId("ChocolateyGUI")
+    name = "Build"
+
+    artifactRules = """
+        code_drop/MsBuild.log
+        code_drop/MSBuild.msi.log
+        code_drop/ChocolateyGUI.msi
+        code_drop/TestResults/issues-report.html
+        code_drop/Packages/**/*.nupkg
+    """.trimIndent()
+
+    params {
+        param("env.vcsroot.branch", "%vcsroot.branch%")
+        param("env.Git_Branch", "%teamcity.build.vcs.branch.ChocolateyGUI_ChocolateyGuiVcsRoot%")
+        param("teamcity.git.fetchAllHeads", "true")
+        password("env.TRANSIFEX_API_TOKEN", "credentialsJSON:c81283e6-cf59-5c9e-9766-6f465018a295", display = ParameterDisplay.HIDDEN, readOnly = true)
+        password("env.GITHUB_PAT", "credentialsJSON:77847f9b-fb3c-59cb-be70-a708b3d3d1ed", display = ParameterDisplay.HIDDEN, readOnly = true)
+    }
+
+    vcs {
+        root(DslContext.settingsRoot)
+
+        branchFilter = """
+            +:*
+        """.trimIndent()
+    }
+
+    steps {
+        powerShell {
+            name = "Prerequisites"
+            scriptMode = script {
+                content = """
+                    # Install Chocolatey Requirements
+                    if ((Get-WindowsFeature -Name NET-Framework-Features).InstallState -ne 'Installed') {
+                        Install-WindowsFeature -Name NET-Framework-Features
+                    }
+
+                    choco install visualstudio2019buildtools netfx-4.8-devpack --confirm --no-progress
+                    exit ${'$'}LastExitCode
+                """.trimIndent()
+            }
+        }
+
+        step {
+            name = "Include Signing Keys"
+            type = "PrepareSigningEnvironment"
+        }
+
+        script {
+            name = "Call Cake"
+            scriptContent = "call build.official.bat"
+        }
+    }
+
+    triggers {
+        vcs {
+            branchFilter = ""
+        }
+    }
+
+    features {
+        pullRequests {
+            vcsRootExtId = "${DslContext.settingsRoot}"
+            provider = github {
+                authType = token {
+                    token = "%env.GITHUB_PAT%"
```

Comment:
```suggestion
                    token = "%system.GitHubPAT%"
```

Reply:
Sorry, I'm jumping into this without contex of any discussions that've happened outside of the PR.

It seems like the intention here was to use the `env.GITHUB_PAT` parameter set above. If it's not being used down here, is the parameter still needed above (i.e. is it used as an environment variable in the build)?

Reply:
You're bang on - Gary mentioned that I should specify the variable name to use, and he'd remove the above.

Reply:
That's alright- I think / suspect what we can do here, then (though will need to test this) that you should be able to set a parameter `env.GitLab_PAT` to the value of `%system.GitLabPAT%`. Will that work?

Reply:
Ok, cool, let's take this for a spin! :shipit: